test/upgrade/test-upgrade.bats

```diff
@@ -212,6 +209,25 @@ failed    | exited     | 17
     done < <(parse_table "$tests")
 }
 
+@test "network - curl" {
+    run curl --max-time 3 -s 127.0.0.1:$HOST_PORT/index.txt
+    is "$output" "$RANDOM_STRING_1" "curl on running container"
+    # restart the container and check if we can still use the port
+    run_podman stop -t0 myrunningcontainer
+    run_podman start myrunningcontainer
+    run curl --max-time 3 -s 127.0.0.1:$HOST_PORT/index.txt
+    is "$output" "$RANDOM_STRING_1" "curl on running container"
+}
+
+@test "network - connect" {
+    skip_if_version_older 2.2.0 "network dis/connect needs v2.2.0"
```

Comment:
If you accept my suggestion above, you can remove the second argument here. I think that would be cleaner.